tests/MongoDB.Driver.Core.Tests/Core/Servers/ServerTests.cs

```diff
@@ -246,4 +253,93 @@ public void A_description_changed_event_with_a_heartbeat_exception_should_clear_
             _mockConnectionPool.Verify(p => p.Clear(), Times.Once);
         }
     }
+
+    public class ServerChannelTests
+    {
+        [SkippableTheory]
+        [InlineData(1, 2, 2)]
+        [InlineData(2, 1, 2)]
+        public void Command_should_send_the_greater_of_the_session_and_cluster_cluster_times(long sessionTimestamp, long clusterTimestamp, long expectedTimestamp)
```

Comment:
This test is a pain to set up. The idea is to send the server a ping command and assert that the driver sent the higher of the two cluster times in the actual command sent to the server.

Note that because we can't sign the cluster times the server will return an error, but that doesn't matter for the purposes of this test.

I verified that this test failed before writing the fix.